java/ql/lib/semmle/code/java/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3136,9 +3178,66 @@ private class PathNodeSink extends PathNodeImpl, TPathNodeSink {
 
   override Configuration getConfiguration() { result = config }
 
-  override PathNodeImpl getASuccessorImpl() { none() }
+  override PathNodeImpl getASuccessorImpl() {
+    result = TPathNodeSinkGroup(this.getSinkGroup(), config)
+  }
 
   override predicate isSource() { sourceNode(node, state, config) }
+
+  string getSinkGroup() { config.sinkGrouping(node.asNode(), result) }
```

Comment:
Should this be `getASinkGroup` since there might be 0 or 1 (or maybe even multiple?) results?

Reply:
I don't think so. Expecting 0 or 1 result is, I think, a reasonable expectation, so the name shouldn't indicate that 2+ is expected.